### PR TITLE
fixed wrong enum value for interac payment network

### PR DESCRIPTION
**Pull request by michaelgobbers**

### CODE REVIEW

**Comment by michaelgobbers** [comment]
This should fix #46

**Comment by mit-73** [comment]
Thanks for the PR. 
Could you redirect it to the `dev` branch?
You also need to raise the versions in pubspec and describe the CHANGELOG of all the packages you have affected.

File: packages/mad_pay_platform_interface/lib/src/models/apple_parameters.dart [hunk 1]
```diff
@@ -84,7 +84,7 @@ enum MerchantCapabilities {
   emv,
 }
 
-/// MerchantCapabilities extension
+/// MerchantCapabilities extension/// MerchantCapabilities extension
```
Comment (mit-73) [inline packages/mad_pay_platform_interface/lib/src/models/apple_parameters.dart:87]: Duplicate

**Comment by michaelgobbers** [comment]
@mit-73 bumped versions, resolved comments and added changelog entries

**Comment by mit-73** [comment]
@michaelgobbers thanks for the PR, released a [new version](https://pub.dev/packages/mad_pay/versions/2.2.10) in pub.

**Pull request merged**